react/containers/ChannelClaimsDisplay/index.js

```diff
@@ -0,0 +1,27 @@
+import { connect } from 'react-redux';
+import { updateChannelClaimsData } from 'actions/show';
+import View from './view';
+
+const mapStateToProps = ({ show }) => {
+  return {
+    name       : show.showChannel.channelData.name,
```

Comment:
They will always exist when ChannelClaimsDisplay is rendered, unless that should change... I am updated it to be destructured, is that what you were thinking?

Reply:
I was just wondering if there would ever be a case when `show` is undefined. Which would cause an errror. `cannot read property 'showChannel` of undefined`.

Or if any of those children would be undefined which would throw an error